lib/git_worktree.rb

```diff
@@ -356,7 +392,7 @@ def current_index
   end
 
   def create_commit(message, tree, parents)
-    author = {:email => @email, :name => @username, :time => Time.now}
+    author = {:email => @email, :name => @username || @email, :time => Time.now}
```

Comment:
Why add this change?

Reply:
The original code has username defaulted as blank in initialize, so now that I've changed it to nil, I've made email a backup.  This class needs a lot of work, because even email can be nil, but this at least gives a new sensible default.